addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -228,12 +228,16 @@ def _get_operator(self, previous_operator_id=None, lang=None, country_id=None):
         if not self.available_operator_ids:
             return False
         self.env.cr.execute("""
+            WITH operator_rtc_session AS (
+                SELECT s.id, member.partner_id as partner_id
+                  FROM discuss_channel_rtc_session s
+                  JOIN discuss_channel_member member ON (member.id = s.channel_member_id)
```

Comment:
```suggestion
                SELECT COUNT(DISTINCT s.id) as nbr, member.partner_id as partner_id
                  FROM discuss_channel_rtc_session s
                  JOIN discuss_channel_member member ON (member.id = s.channel_member_id)
                  GROUP BY member.partner_id
```
and update subsequent query?